### DISCUSSION TITLE
Severe problem in add album?

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#327] [Fixed Issues]
@rern 

Since yesterday's update (20210515) I got for the second time an unresponsible system:
Adding any local folder (an album) to an empty playlist is adding 16384 entries (I think the first 2^14 files from the NAS). cifsd stays top of process list for a long time. Increasing max_output_buffer_size to 32768 did not solve the problem.
System is unusable.

EDIT: I re-installed the 20210507 release.

**Comment by ceejayemm** [comment]
I am seeing a slight variation on this problem (also on rAudio1 (20210515) 5.10.25-4-ARCH aarch64. My Library is NFS mounted from my NAS. If I add an album (via Replace or Replace Play), the Playlist is emptied and nothing added. However if I add a Webradio (also via Replace or Replace/Play), then the web radio station IS added to the Playlist.

Chris

**Reply by ceejayemm** [reply]
Further to the above, if I add an album after selecting an Artist first, then the album IS added to the Playlist.

Chris

**Reply by AndyFx12** [reply]
> 
> 
> I am seeing a slight variation on this problem (also on rAudio1 (20210515) 5.10.25-4-ARCH aarch64. My Library is NFS mounted from my NAS. If I add an album (via Replace or Replace Play), the Playlist is emptied and nothing added. However if I add a Webradio (also via Replace or Replace/Play), then the web radio station IS added to the Playlist.
> 
> Chris

I think "nothing added" is wrong - in my case, whole library is added to playlist. It take some time to display them ....

Andy

**Reply by findnix** [reply]
I can confirm this: the whole library is added.

**Reply by klassikmann** [reply]
> I can confirm this: the whole library is added.

In my mpd database are about 18.000 tracks. Not the whole database is added, but exactly 16.384 (= 2^14).

**Reply by ceejayemm** [reply]
Tried this again last night.  My NAS has around 315 albums and 4100+ tracks, so not big in comparison to some others :-).  Cleared the Playlist, selected an album (a folder on my NAS) from my Library and used Replace. I left the box overnight and NOTHING has been added to my Playlist this morning. So my case does seem slightly different for some reason, not sure why.

Chris

**Comment by AndyFx12** [comment]
Seems to be related to contextmenu of album coverart. Contextmenu of listview at library works on my setup (NFS).

Andy

**Comment by findnix** [comment]
Hi @rern !
Updated via "UPDATE" Channel: Problem fixed.
THANK YOU

**Reply by ceejayemm** [reply]
UPDATE Branch works for me too.

Chris

**Reply by AndyFx12** [reply]
> Hi @rern !
> Updated via "UPDATE" Channel: Problem fixed.
> THANK YOU

Same here, thanks!

Andy

**Reply by klassikmann** [reply]
First tests very promising.
